templates/admission-policy.yaml

```diff
@@ -0,0 +1,56 @@
+{{- $kubeVersion := semverCompare ">1.30.0" .Values.global.clusterConfiguration.kubernetesVersion }}
+apiVersion: admissionregistration.k8s.io/{{ if $kubeVersion }}v1{{ else }}v1alpha1{{ end }}
```

Comment:
"admissionregistration.k8s.io/v1 for >1.30" seems strange. v1 appears in 1.30. 1.28 and 1.29 contains v1beta1 version for ValidatingAdmissionPolicy and ValidatingAdmissionPolicyBinding  (changelog recommends migrate from v1alpha to v1beta when upgrading from 1.27 to 1.28)